src/test/java/bit/javaoop/FixedBudgetManagerTest.java

```diff
@@ -0,0 +1,47 @@
+package bit.javaoop;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
```

Comment:
Tutaj jakieśnie używane importy ctrl + alt + O to skrót do optymalizacji importów warto sobie wyrobić taki nawyk combo (ctrl + alt + L do tego ctrl + alt + O)